packages/react-error-overlay/src/containers/StackFrame.js

```diff
@@ -150,7 +154,9 @@ class StackFrame extends Component {
         </div>
         {codeBlockProps &&
           <span>
-            <CodeBlock {...codeBlockProps} />
+            <a onClick={this.openInEditor} style={codeAnchorStyle}>
```

Comment:
This should only be done if `openInEditor` is available (i.e. if conditions inside `openInEditor` are `true`). We shouldn't see a clickable cursor when it's not actually available.

Reply:
Now that I think of it maybe it doesn't matter. The only case in which I can imagine this happening is when Webpack initialization code itself throws (and we're left with webpack bootstrap code with no app frames). But maybe that's too rare to count.